src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/request/RaptorRequestTransferCache.java

```diff
@@ -35,14 +35,20 @@ public LoadingCache<CacheKey, RaptorTransferIndex> getTransferCache() {
     return transferCache;
   }
 
+  public void put(List<List<Transfer>> transfersByStopIndex, RouteRequest request) {
+    final CacheKey cacheKey = new CacheKey(transfersByStopIndex, request);
+    final RaptorTransferIndex raptorTransferIndex = RaptorTransferIndex.create(
+      transfersByStopIndex,
+      cacheKey.request
+    );
+
+    LOG.info("Adding request from config to cache: {}", cacheKey.options);
```

Comment:
"config" is a pretty general term. Maybe this would be clearer: 

```suggestion
    LOG.info("Initializing cache with request: {}", cacheKey.options);
```